packages/playwright-ct-core/src/vitePlugin.ts

```diff
@@ -153,6 +143,19 @@ export function createPlugin(
         sourcemap: true,
       };
 
+      const frameworkViteConfig = frameworkViteConfigFn && frameworkViteConfigFn();
+      if (frameworkViteConfig && viteConfig.plugins?.length)
+        delete frameworkViteConfig.plugins;
+
+      if (frameworkViteConfig)
+        viteConfig = mergeConfig(viteConfig, frameworkViteConfig);
+
+      viteConfig.plugins ??= [];
```

Comment:
I think at this point we know for sure that the plugins are there.

Reply:
`viteConfig.plugins` could be `undefined` when there is no `frameworkViteConfig` is specified.